modules/services/mpd-discord-rpc.nix

```diff
@@ -0,0 +1,46 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.mpd-discord-rpc;
+  tomlFormat = pkgs.formats.toml { };
+  configFile = tomlFormat.generate "config.toml" cfg.settings;
+in {
+  meta.maintainers = [ maintainers.kranzes ];
+
+  options.services.mpd-discord-rpc = {
+    enable = mkEnableOption "the mpd-discord-rpc service";
+
+    settings = mkOption {
+      type = tomlFormat.type;
+      default = { };
+      example = {
+        hosts = [ "localhost:6600" ];
+        format = {
+          details = "$title";
+          state = "On $album by $artist";
+        };
+      };
+      description = ''
+        Configuration included in <literal>config.toml</literal>.
+        For available options see <link xlink:href="https://github.com/JakeStanger/mpd-discord-rpc#configuration"/> 
+      '';
+    };
+  };
+
+  config = mkIf cfg.enable {
+    xdg.configFile."discord-rpc/config.toml".source = configFile;
+
+    systemd.user.services.mpd-discord-rpc = {
+      Unit = {
+        Description = "Discord Rich Presence for MPD";
+        Documentation = "https://github.com/JakeStanger/mpd-discord-rpc";
+        After = [ "graphical-session-pre.target" ];
+        PartOf = [ "graphical-session.desktop" ];
+      };
+      Service.ExecStart = "${pkgs.mpd-discord-rpc}/bin/mpd-discord-rpc";
```

Comment:
We try to make the package configurable when possible in Home Manager.
Could you add a `package` option?
As an example: https://github.com/nix-community/home-manager/blob/d119cea3763977801ad66330668c1ab4346cb7f7/modules/programs/jq.nix#L33-L38

Reply:
done.